yarn.lock

```diff
@@ -1784,7 +1784,7 @@
     "@types/websocket" "^1.0.3"
     ws "^8.14.2"
 
-"@supabase/storage-js@^2.5.4":
+"@supabase/storage-js@2.5.5", "@supabase/storage-js@^2.5.4":
```

Comment:
To help with this, I think we should change all `@supabase/` packages to accept patch-level upgrades via `~`

Reply:
done